ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx

```diff
@@ -200,10 +200,65 @@ function NetworkGraphPage() {
                 .then((values) => {
                     // get policy nodes from policy graph API response
                     const { nodes: policyNodes } = values[1].response;
-                    // transform policy data to DataModel
-                    const { policyDataModel, policyNodeMap } = transformPolicyData(policyNodes);
                     // get active nodes from network flow graph API response
                     const { nodes: activeNodes } = values[0].response;
+
+                    /*
+                    FIX for 4.3.x central <-> 4.4.x sensor mismatch that causes the Network Graph to not render
+
+                    This is a patch to fix a compatibility issue between central and sensor, where sensor sends an entity
+                    type of `5`, which is not understood by central and passed to the UI directly as a numeric value
+                    instead of being converted to a string enum of `INTERNAL_ENTITIES` as it is in central >= 4.4.0
+
+                    The fix is to convert all of these numeric value types to the external entity type of `INTERNET` immediately when
```

Comment:
This location is the only place where the service call to `fetchNetworkFlowGraph` is used.